Aztec/Classes/GUI/FormatBar/FormatBarDelegate.swift

```diff
@@ -10,7 +10,7 @@ public protocol FormatBarDelegate : NSObjectProtocol {
     /// Prompts the delegate that the bar item with the specified identifier was tapped,
     /// and it should take appropriate action.
     ///
-    func handleActionForIdentifier(_ identifier: FormattingIdentifier, barItem: FormatBarItem)
+    func handleActionForIdentifier(_ identifier: String, barItem: FormatBarItem)
```

Comment:
Even though this wasn't introduced by this PR specifically, it could be a good opportunity to do some cleanup.

If the `FormatBarItem` instance contains its own identifier, there's no need to specify it as an additional parameter.